src/index.ls

```diff
@@ -60,9 +66,6 @@ exports <<<
   # Same as `tokens`, except that this skips rewriting.
   lex: -> lexer.lex it, {+raw}
 
-  # Runs LiveScript code directly.
-  run: (code, options) -> do Function exports.compile code, {...options, +bare}
-
```

Comment:
Because it was being overwritten, before being called. So I removed it to get your attention, it worked. 😄
Any hints on how to improve it? It's being overwritten by `lib/node`.